homeassistant/components/elmax/manifest.json

```diff
@@ -6,5 +6,10 @@
   "documentation": "https://www.home-assistant.io/integrations/elmax",
   "iot_class": "cloud_polling",
   "loggers": ["elmax_api"],
-  "requirements": ["elmax-api==0.0.4"]
+  "requirements": ["elmax_api==0.0.4"],
```

Comment:
That's because a wrong copy-paste. In any case, hyphens and underscore are normalized in pipy, so both statements are equivalent. I see your point tough, so I've updated it in 7569ced5e39a4837cfb8a476a7e2bfa454b42799